CustomInvite/src/main/res/values/strings.xml

```diff
@@ -13,13 +17,13 @@
     <string name="custom_invite_1">Custom Invite #1</string>
     <string name="custom_invite_2">Custom Invite #2</string>
     <string name="set_contact_details_launch_invite">Set contact details &amp; launch invite</string>
-    <string name="reset_state">Reset state</string>
+    <string name="reset_state">Reset State</string>
     <string name="description_default">This example demonstrates how you can supply your user\'s contact details to speed up the invite process, and also use the default invite listener interface to be informed of any events involving the invitation and survey process</string>
     <string name="description_custom1">This example shows how to use the custom invite framework to include an in-line prompt to receive an email or SMS survey. Enter contact details in the text field and click the button to receive a sample survey.</string>
     <string name="description_custom2">This example shows a simple snackbar implementation of the invite process. Click \'Launch invitation\' to see a custom invitation appear at the bottom of the screen</string>
     <string name="FORESEE.contactDetailsEmptyInputError">Enter a mobile number or email address</string>
     <string name="FORESEE.contactDetailsInvalidInputError">Format should be: name@domain.com or 123-456-7890</string>
     <string name="reset_button_desc">Once the invite is shown, the SDK drops into an idle state until the repeat days have elapsed. Click here to reset the state of the SDK.</string>
-    <string name="main_screen_desc">This sample demonstrates some simple custom invitation styles</string>
+    <string name="main_screen_desc">This sample demonstrates some simple custom invitation styles. Select one of the \"Custom Invite\" button to show an invite. The SDK transitions to an idle state after an invite is displayed. Use "Reset State" to test again. (This will also delete pre-set contact details.). Internet connection is required.</string>
```

Comment:
Minor grammar issues here. I'm going to go ahead and make a couple small updates to avoid a back-and-forth.